api/src/Page/Shipment.php

```diff
@@ -1226,12 +1226,14 @@ function _dispatch_dewar()
                 } else {
                     try {
                         $shipment_id = $this->_dispatch_dewar_in_shipping_service($data, $dew);
+                        if (Utils::getValueOrDefault($shipping_service_links_in_emails)) {
+                            $data['AWBURL'] = $this->shipping_service->get_awb_pdf_url($shipment_id);
+                        }
                     } catch (Exception $e) {
-                        error_log($e);
-                        $data['AWBURL'] = "";
-                    }
-                    if (Utils::getValueOrDefault($shipping_service_links_in_emails)) {
-                        $data['AWBURL'] = $this->shipping_service->get_awb_pdf_url($shipment_id);
+                        $error_json = json_decode($e->getMessage());
+                        $error_response = $error_json->content->detail;
```

Comment:
```suggestion
                        $error_response = $error_json->content->detail ?? $e->getMessage();
```
As at line 1219 (just in case)